modules/claims_api/spec/workers/poa_form_builder_job_spec.rb

```diff
@@ -29,7 +30,7 @@
         },
         phone: {
           areaCode: '555',
-          ohoneNumber: '5551337'
+          phoneNumber: '5551337'
```

Comment:
this, and the similar spelling mistake below might be my favorite thing about this pr.
i spent far too long trying to understand why this information wasn't being populated into the pdf before i discovered this 😞